### ISSUE TITLE
Font size issues

### ISSUE THREAD

**Issue by josephbona** [TrellisCommerce/shopify-tailwind-starter-base#4]
**Describe the current behavior**
Tailwind is inheriting the `html` font-size from Dawn, which is 6.25px by default.

**Describe the expected behavior**
Ability to set a base font size that works with Tailwind but does not affect Dawn styles.

**Version information (Dawn, browsers and operating systems)**
- Dawn Version: 1.0.0
- Chrome Version 91.0.4472.114
- macOS Version 11.3.1

**Possible solution**

**Additional context/screenshots**
Add any other context about the problem here. If applicable, add screenshots to help explain.

**Comment by ineptian** [comment]
I agree this is weird. This probably isnt the greatest solution but it kind of works.

You can modify [this solution](https://github.com/tailwindlabs/tailwindcss/issues/1232#issuecomment-1150725542) to work for this. Basically just multiply the output rems by 1.6. I have pasted the modified code below.

Paste the following above module.exports = { in your tailwind.config.js

```
const defaultTheme = require("tailwindcss/defaultTheme");

function rem2rem(input, fontSize = 1.6) {
  if (input == null) {
    return input;
  }
  switch (typeof input) {
    case "object":
      if (Array.isArray(input)) {
        return input.map((val) => rem2rem(val, fontSize));
      } else {
        const ret = {};
        for (const key in input) {
          ret[key] = rem2rem(input[key]);
        }
        return ret;
      }
    case "string":
      return input.replace(
        /(\d*\.?\d+)rem$/,
        (_, val) => parseFloat(val) * fontSize + "rem"
      );
    default:
      return input;
  }
}
```
And then within the area where you define all your tailwind theme settings ```theme {}``` add:
```
    borderRadius: rem2rem(defaultTheme.borderRadius),
    columns: rem2rem(defaultTheme.columns),
    fontSize: rem2rem(defaultTheme.fontSize),
    lineHeight: rem2rem(defaultTheme.lineHeight),
    spacing: rem2rem(defaultTheme.spacing),
    maxWidth: ({ theme, breakpoints }) => ({
      ...rem2rem(defaultTheme.maxWidth({ theme, breakpoints })),
    }),
```
You can add it to all of the utilities you need.

Probably not the best solution but a dirty fix for now.

**Comment by cmegalo** [comment]
@josephbona it looks like the base font-size of Dawn is now 10px, which is definitely more user-friendly, but here is what I would recommend for setting base sizes that work with Tailwind and do not affect Dawn styles:

For spacing & font sizes, override values in tailwind.config.js like so (for a 16px scale):
```javascript
...
theme: {
    spacing: {
      1: '4px',
      2: '8px',
      3: '12px',
      4: '16px',
      5: '20px',
      6: '24px',
      ...
    },
    fontSize: {
      ...
      base: '16px',
      ...
    },
},
...
```
